app/services/doc_auth/lexis_nexis/config.rb

```diff
@@ -5,8 +5,6 @@ module LexisNexis
       :base_url, # required
       :request_mode,
       :trueid_account_id,
-      :trueid_liveness_cropping_workflow,
-      :trueid_liveness_nocropping_workflow,
       :trueid_noliveness_cropping_workflow,
```

Comment:
these configs are still needed for acuant. should we rename them since liveness is taken out?

Reply:
If these are used for Acuant, ideally we'll be getting rid of them when we move on to taking selfie out of that. It may not be worth renaming them if we are going to remove them.